packages/console/src/console.js

```diff
@@ -1,8 +1,8 @@
 // @ts-check
 
-import { ErrorInfo } from '@agoric/assert';
+import { ErrorInfo } from '@agoric/assert/src/assert';
+import '@agoric/assert/exported';
 
-// @ts-ignore fromEntries missing from Object type
```

Comment:
How did this problem go away?

Reply:
I added a `jsconfig.json`.

Reply:
... and specified `"esnext"` as the Typescript target.